metrics/web/test/features/auth/domain/usercases/receive_current_user_updates_test.dart

```diff
@@ -11,5 +13,19 @@ void main() {
         MatcherUtil.throwsAssertionError,
       );
     });
+
+    test("provides a stream that emmits user updates", () async {
+      final seedUser = User(id: 'seed');
+
+      final repository = UserRepositoryStub(seedUser: seedUser);
+      final receiveUserUpdates = ReceiveCurrentUserUpdates(repository);
```

Comment:
Here I would also use same approach as presented in SignIn test: check if call performed or if any exceptions can be thrown from this use case, we shouldn't test stub here.